casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -314,4 +318,61 @@ object RuntimeManager {
       _                = assert(hash == replayHash)
       runtime          <- MVar[F].of(active)
     } yield new RuntimeManagerImpl(hash, runtime)
+
+  def forTrans[F[_]: Monad, T[_[_], _]: MonadTrans](
+      runtimeManager: RuntimeManager[F]
+  ): RuntimeManager[T[F, ?]] =
+    new RuntimeManager[T[F, ?]] {
+      override def captureResults(
+          start: RuntimeManager.StateHash,
+          deploy: Deploy,
+          name: String
+      ): T[F, scala.Seq[Par]] = runtimeManager.captureResults(start, deploy, name).liftM[T]
+
+      override def captureResults(
+          start: RuntimeManager.StateHash,
+          deploy: Deploy,
+          name: Par
+      ): T[F, scala.Seq[Par]] = runtimeManager.captureResults(start, deploy, name).liftM[T]
+
+      override def replayComputeState(
+          hash: RuntimeManager.StateHash,
+          terms: scala.Seq[InternalProcessedDeploy],
+          time: Option[Long]
+      ): T[F, scala.Either[(Option[Deploy], Failed), RuntimeManager.StateHash]] =
+        runtimeManager.replayComputeState(hash, terms, time).liftM[T]
+
+      override def computeState(
+          hash: RuntimeManager.StateHash,
+          terms: scala.Seq[Deploy],
+          time: Option[Long]
+      ): T[F, (RuntimeManager.StateHash, scala.Seq[InternalProcessedDeploy])] =
+        runtimeManager.computeState(hash, terms, time).liftM[T]
+
+      override def storageRepr(
+          hash: RuntimeManager.StateHash
+      ): T[F, Option[String]] = runtimeManager.storageRepr(hash).liftM[T]
+
+      override def computeBonds(
+          hash: RuntimeManager.StateHash
+      ): T[F, scala.Seq[Bond]] = runtimeManager.computeBonds(hash).liftM[T]
+
+      override def getData(
+          hash: ByteString,
+          channel: Par
+      ): T[F, scala.Seq[Par]] = runtimeManager.getData(hash, channel).liftM[T]
+
+      override def getContinuation(
+          hash: ByteString,
+          channels: scala.collection.immutable.Seq[Par]
+      ): T[F, scala.Seq[(scala.Seq[BindPattern], Par)]] =
+        runtimeManager.getContinuation(hash, channels).liftM[T]
+
+      override def emptyStateHash: ByteString = runtimeManager.emptyStateHash
```

Comment:
So you don't need to lift this one because it is a field?

Reply:
yup.

Reply:
maybe a better answer would be: it's a constant.

Reply:
I can replace the def with a val.